### PR TITLE
Grammar fixes.

### PR DESCRIPTION
**Pull request by Iangit1**
## Pull request type

- [ ] Feature addition
- [ ] Bug fix
- [x] Grammar or minor corrections

## Does it overwrites current functionalities?
No

- [ ] Yes
- [x] No

## Did you test all of your changes?

- [ ] Yes
- [x] No
but just grammar
## Description of the changes

Fixed grammatical errors.

### CODE REVIEW

File: warnsystem/warnsystem.py [hunk 1]
```diff
@@ -455,9 +455,9 @@ def _set_log(self, sentry: "Log"):
         """
         Set if the bot should respect roles hierarchy.
 
-        If enabled, a member cannot ban another member above him in the roles hierarchy, like\
+        If enabled, a member cannot ban another member above them in the roles hierarchy, like\
```
Comment (retke) [inline warnsystem/warnsystem.py]: "a member" is singular here, "him" should stay.

File: warnsystem/warnsystem.py [hunk 2]
```diff
@@ -976,7 +976,7 @@ def _set_log(self, sentry: "Log"):
         """
         Softban the member from the server.
 
-        This means that the user will be banned and immediately unbanned, so it will purge his\
+        This means that the user will be banned and immediately unbanned, so it will purge their\
```
Comment (retke) [inline warnsystem/warnsystem.py]: "the user" is also singular here.

File: warnsystem/warnsystem.py [hunk 3]
```diff
@@ -1012,7 +1012,7 @@ def _set_log(self, sentry: "Log"):
         - `[p]warn 5 @user`: Ban for no reason :c
         - `[p]warn 5 @user 7d Insults`: 7 days ban for the reason "Insults"
         - `[p]warn 5 012345678987654321 Advertising and leave`: Ban the user with the ID provided\
-        while he's not in the server for the reason "Advertising and leave" (if the user shares\
+        while they're not in the server for the reason "Advertising and leave" (if the user shares\
```
Comment (retke) [inline warnsystem/warnsystem.py]: "Ban the user" also singular.

File: warnsystem/warnsystem.py (hunk 1 above)
Comment (Iangit1) [reply warnsystem/warnsystem.py]: English is an annoying language. "Them" can be be used as plural or singular depending on how it's used.

File: warnsystem/warnsystem.py (hunk 2 above)
Comment (Iangit1) [reply warnsystem/warnsystem.py]: Again, their can be used as plural or singular.

File: warnsystem/warnsystem.py (hunk 3 above)
Comment (Iangit1) [reply warnsystem/warnsystem.py]: And again, their can be used as plural or singular.

**Pull request merged**